spiceypy/spiceypy.py

```diff
@@ -15859,3 +15905,27 @@ def xposeg(matrix, nrow, ncol):
     nrow = ctypes.c_int(nrow)
     libspice.xposeg_c(matrix, nrow, ncol, mout)
     return stypes.cMatrixToNumpy(mout)
+
+
+@spiceErrorCheck
+def zzdynrot(typid, center, et):
+    """
+    Find the rotation from a dynamic frame ID to the associated frame at the time requested
+
+    :param typid: ID code for the dynamic frame
+    :type typid: int
+    :param center: the ID for the center of the frame
+    :type center: int
+    :param et: Epoch measured in seconds past J2000
+    :type et: float
+    :return:  Rotation matrix from the input frame to the returned associated frame, id for the associated frame
+    :rtype: tuple
+    """
+    typid = ctypes.c_int(typid)
+    center = ctypes.c_int(center)
+    et = ctypes.c_double(et)
+    matrix = stypes.emptyDoubleMatrix(x=3, y=3)
+    nextFrame = ctypes.c_int()
+    libspice.zzdynrot_(ctypes.byref(typid), ctypes.byref(center), ctypes.byref(et), matrix, ctypes.byref(nextFrame))
+    return stypes.cMatrixToNumpy(matrix), nextFrame.value
```

Comment:
by convention, I explicitly cast return code values to booleans in wrapper functions ie `bool(x.value)`, please update here and fro tkfram and ckfrot

Reply:
I updated `tkfram` and `ckfrot`, but `zzdynrot` doesn't return a found variable so I don't think there was anything to update for it?